src/main/java/edu/harvard/iq/dataverse/export/ddi/DdiExportUtil.java

```diff
@@ -313,15 +315,36 @@ private static void writeDataAccess(XMLStreamWriter xmlw , DatasetVersionDTO ver
         XmlWriterUtil.writeFullElement(xmlw, "conditions", version.getConditions());
         XmlWriterUtil.writeFullElement(xmlw, "disclaimer", version.getDisclaimer());
         xmlw.writeEndElement(); //useStmt
-        
+
         /* any <note>s: */
+        if (version.getTermsOfUse() != null && !version.getTermsOfUse().trim().equals("")) {
+            xmlw.writeStartElement("notes");
+            xmlw.writeAttribute("type", NOTE_TYPE_TERMS_OF_USE);
+            xmlw.writeAttribute("level", LEVEL_DV);
+            xmlw.writeCharacters(version.getTermsOfUse());
+            xmlw.writeEndElement(); //notes
+        }
+
```

Comment:
Can you add to DdiExportUtilTest.java to exercise this code?

Reply:
Added dataset-finch-terms-of-use.json and dataset-finch-terms-of-use.xml which is used in testJson2DdiNoFilesTermsOfUse() for custom terms of use